src/main/java/uk/gov/digital/ho/hocs/workflow/client/caseworkclient/CaseworkClient.java

```diff
@@ -281,4 +283,8 @@ public void mapCaseData(UUID caseUUID, Map<String, String> keyMappings) {
         log.info("Data for caseUUID: {} mapped successfully.", caseUUID);
     }
 
+    public void createCaseTag(UUID caseUUID, String tag) {
+        CaseTagRequest caseTagRequest = new CaseTagRequest(tag);
+        restHelper.post(serviceBaseURL, String.format("/case/%s/tag", caseUUID), caseTagRequest, String.class);
```

Comment:
```suggestion
        restHelper.post(serviceBaseURL, String.format("/case/%s/tag", caseUUID), caseTagRequest, Void.class);
```

We're not utilising the return here so I don't think we want to try and convert it...

Reply:
Incorporated changes.